pkg/server/interfaces/api/authentication.go

```diff
@@ -83,6 +83,22 @@ func (c *authentication) GetWebServiceRoute() *restful.WebService {
 		Returns(200, "", apis.LoginUserInfoResponse{}).
 		Returns(400, "", bcode.Bcode{}).
 		Writes(apis.LoginUserInfoResponse{}))
+
+	ws.Route(ws.GET("/admin_configured").To(c.adminConfigured).
+		Doc("check admin is configured").
+		Metadata(restfulspec.KeyOpenAPITags, tags).
+		Returns(200, "", apis.AdminConfiguredResponse{}).
+		Returns(400, "", bcode.Bcode{}).
+		Writes(apis.AdminConfiguredResponse{}))
+
+	ws.Route(ws.PUT("/init_admin").To(c.configureAdmin).
```

Comment:
This API definition misses the permission check. So, it is dangerous. We must ensure this API only be called when there are no users.

Reply:
Yes, there are checks in the `InitAdmin`.